src/os/OS.hxx

```diff
@@ -741,6 +741,7 @@ extern unsigned critical_nesting;
 extern uint32_t SystemCoreClock;
 }
 #define cm3_cpu_clock_hz SystemCoreClock
+#define cpu_clock_hz SystemCoreClock
```

Comment:
should these defines and the above extern block be below the !defined(ESP32) below? I don't see any usages of these in the ESP32 code paths. There likely isn't harm in having them here.

Reply:
done